autoload/lsp/lspserver.vim

```diff
@@ -155,12 +155,12 @@ def InitServer(lspserver: dict<any>, bnr: number)
   # Compute the rootpath (based on the directory of the buffer)
   var rootPath = ''
   var rootSearchFiles = lspserver.rootSearchFiles
-  var bufDir = bnr->bufname()->fnamemodify(':p:h')
+  var bufDir = bnr->bufname()->resolve()->fnamemodify(':p:h')
```

Comment:
If you resolve a symbolic link in Unix, then the root search will find the wrong root directory.  The symbolic link should not be resolved.

Reply:
it did with `cwd` together, it should be fine, which just to avoid one is symblink, another was not, then it would be wrong.

Reply:
or whatever, now removed it.